### PR TITLE
docs: Update to include guide for OpenAI LLMs

### PR DESCRIPTION
**Pull request by amanape**
**What is the problem that this fixes or functionality that this introduces? Does it fix any open issues?**
There was no guide for OpenAI LLMs

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

---
**Other references**

### CODE REVIEW

File: docs/modules/usage/llms/openai-llms.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+From there, you can set your model and API key in the settings window.
+
+## Using OpenAI-Compatible Endpoints
+
+Just as for OpenAI Chat completions, we use LiteLLM for OpenAI-compatible endpoints. You can find their full documentation on this topic [here](https://docs.litellm.ai/docs/providers/openai_compatible).
+
+When running the OpenHands Docker image, you'll need to set the following environment variables:
+
+```sh
+LLM_BASE_URL="<api-base-url>" # e.g. "http://0.0.0.0:4000"
```
Comment (tobitege) [inline docs/modules/usage/llms/openai-llms.md:52]: "4000" should be "3000"

File: docs/modules/usage/llms/llms.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ The following environment variables might be necessary for some LLMs/providers:
 
 We have a few guides for running OpenHands with specific model providers:
 
+- [OpenAI](llms/openai-llms)
 - [ollama](llms/local-llms)
 - [Azure](llms/azure-llms)
+- [Google](llms/google-llms)
```
Comment (tobitege) [inline docs/modules/usage/llms/llms.md:32]: Should we do this alphabetically?

**Review by tobitege — APPROVED** [review]
LGTM, nice job! Thanks for this!
Just 2 tiny nits.

File: docs/modules/usage/llms/llms.md (hunk 2 above)
Comment (amanape) [reply docs/modules/usage/llms/llms.md:32]: Or maybe based on popularity instead? 🤔

- OpenAI
- Azure
- Google
- ollama

File: docs/modules/usage/llms/openai-llms.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
```
Comment (amanape) [inline docs/modules/usage/llms/openai-llms.md:52]:
```suggestion
LLM_BASE_URL="<api-base-url>" # e.g. "http://0.0.0.0:3000"
```

File: docs/modules/usage/llms/llms.md (hunk 2 above)
Comment (amanape) [reply docs/modules/usage/llms/llms.md:32]: Or maybe based on popularity instead? 🤔

OpenAI
Azure
Google
ollama

Comment (tobitege) [reply docs/modules/usage/llms/llms.md:32]: So what if I said Google was more popular that Azure? 😏

Comment (amanape) [reply docs/modules/usage/llms/llms.md:32]: 🔫

Comment (amanape) [reply docs/modules/usage/llms/llms.md:32]: Then you would have to open a new PR to change the order and hope I approve it

Comment (tobitege) [reply docs/modules/usage/llms/llms.md:32]: Typing about it took more time than to do it. Already approved, was just a nit. 🤗

**Pull request merged**